build/vsts_build.yaml

```diff
@@ -224,6 +224,25 @@ phases:
     inputs:
       TargetFolder: '$(Build.Repository.LocalPath)\\artifacts\\$(NupkgOutputDir)'
 
+  - task: PowerShell@1
+    displayName: "test only : check curr folder to see if there is nuget.exe file"
+    inputs:
+      scriptType: "inlineScript"
+      inlineScript: |
+        pwd
+        cd '$(Build.Repository.LocalPath)\\artifacts'
```

Comment:
Consider using $(VsixPublishDir)

If you look at the output from Verify Assembly Signatures and StrongName you will see that the NuGet.exe in the nupkg is actually verified, at least the one from the nupgk. 
The nuget.exe in the nupkg should be same as the one in the artifacts folder. 

A simple solution might be just to package the mssign.exe

Reply:
Thanks for telling about $(VsixPublishDir) 
I have a question, are we going to verify .exe directly or package the .exe then verify?

Reply:
That's up to you, whatever is easier. 
If you can verify the exe directly, that's perfect.